Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -316,6 +315,7 @@ struct DQEventMixing {
       }
     }
     histNames += "PairsMuonMEPM;PairsMuonMEPP;PairsMuonMEMM;";
+    histNames += "PairsEleMuMEPM;PairsEleMuMEPP;PairsEleMuMEMM;";
```

Comment:
see also the comments i made lower in the code. Since multiple electron cuts are possible, then, when you combine with muons (currently selected with just one cut) you should have one Pairs histogram directory for each electron cut.